bin/si-mcp-server/src/data/schemaAttributes.ts

```diff
@@ -92,8 +90,8 @@ function collectFlatAttributes(
 
     switch (node.propType) {
       case "object": {
-        const hasChildren = Array.isArray(node.children) &&
-          node.children.length > 0;
+        const hasChildren =
```

Comment:
!@$!@#$ format-on-save. I don't know if we have a formatter for these, it's using the rules that we use for our other .ts files. I can change this back if the current style is intentional, just means re-editing a couple files and save-without-format.